sys/include/net/fib.h

```diff
@@ -63,6 +63,11 @@ typedef struct fib_destination_set_entry_t {
 #define FIB_LIFETIME_NO_EXPIRE (0xFFFFFFFFffffffff)
 
 /**
+ * @brief flag to identify if a route was set by RPL
+ */
+#define FIB_FLAG_RPL_ROUTE (1UL << 0)
```

Comment:
maybe we should consider to use an `enum` here, since I feel it could become more handy when more flags come around